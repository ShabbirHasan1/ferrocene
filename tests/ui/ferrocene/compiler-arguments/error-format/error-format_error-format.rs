// check-fail
// compile-flags: --error-format --error-format

fn main() {}

// ferrocene-annotations: um_rustc_error_format
