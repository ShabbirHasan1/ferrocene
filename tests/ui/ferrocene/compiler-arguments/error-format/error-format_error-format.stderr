error: argument for `--error-format` must be `human`, `json` or `short` (instead was `--error-format`)

